### PR TITLE
essential fixes to make the embeded yaml parse

### PR DESCRIPTION
**Pull request by marc-portier**
title says it all

not yet checking up on the functionality at this time
purely syntactical 

and related to our discussion in #35

### CODE REVIEW

File: analysis_directory/parameters_structured.tsv [hunk 1]
```diff
@@ -204,7 +205,7 @@ pandaseqThreads 20
 #
 #=     pandaseqMinlen:
 #=        title: pandaseq minimum length
-#=        description: Sets the minimum length for a sequence, after primers are removed. By default, all sequences are kept. With this option, sequences shorter than desired can be discarded. In case you need to use this parameter, be sure you leave a tab after 'minlen' and set it like this: '-l 80'. If you do not want to use this parameter, make sure there is nothing after 'minlen'.
+#=        description: "Sets the minimum length for a sequence, after primers are removed. By default, all sequences are kept. With this option, sequences shorter than desired can be discarded. In case you need to use this parameter, be sure you leave a tab after 'minlen' and set it like this: '-l 80'. If you do not want to use this parameter, make sure there is nothing after 'minlen'."
```
Comment (hariszaf) [inline analysis_directory/parameters_structured.tsv:208]: Could you replace this description with the following? 

"Sets the minimum length for a sequence, after primers are removed. Sequences shorter than desired will be discarded. By default, all sequences are kept. To do so, you may leave this parameter empty."

**Comment by hariszaf** [comment]
Would you like to keep this PR open for the python script too or would you prefer to merge and open another one ?

File: analysis_directory/parameters_structured.tsv (hunk 1 above)
Comment (marc-portier) [reply analysis_directory/parameters_structured.tsv]: yes, sure, done

but have to say: this is quite unrelated to what I was trying to achieve here and now

* I am only focusing on syntactic yaml parsing of the paramaters file so I can go back to processing it with the external yaml4parms, not on correct semantic descriptions
* my feeling is that you can safely apply these extra descriptions in your own branch & merge at any time convenient -- or else merge the PR first and then add those changes ?) 


I already hooked up a test inside the yaml4parms to the latest paramaters-structured.tsv inside pema, so I automaticallt stay in sync with your updates .

At some time (after actual release of yaml4parms I suppose the reverse hookup will make sense to (some CI/CD action in this pema repo could use the yaml4parms to check)

**Comment by marc-portier** [comment]
> Would you like to keep this PR open for the python script too or would you prefer to merge and open another one ?

the latter: merge asap and close

I hope to follow this scenario:

- not to be needing more PR on pema for now (but we can surely keep discussing optimal semantics through #35 
- tune the external tool yaml4parms so it can see some v0.1.0 release (it should imho produce yaml, json, csv and html-form of the params) 
- then come back to pema and have some simple pema-describe.sh script that uses the yaml4parms in the back 
- and as an extra maybe add a ci/cd action here in the pema repo to have that script check if some commit on the parameters.tsv is not inadvertently introducing bad syntax

**Comment by hariszaf** [comment]
Super! This sounds good to me too! 

I am now merging this PR and many thanks for the insight!

**Pull request merged**